data.py

```diff
@@ -1,26 +1,36 @@
-from sys import stdin
+import os
 
-def collect_matrix():
-    matrix = list()
-    previous_line = ""
-    for line in stdin:
-        if previous_line == "":
-            previous_line = line
-        if len(previous_line.split()) == len(line.split()):
-            matrix.append(line.rstrip("\n"))
-            previous_line = line
-        elif line == "\n":
-            break
-        else:
-            print("warning: the string is not saved. the number of elements is not equal.")
-    return matrix
+def collect_matrices():
+    if os.path.exists("matrices.txt"):
+        file = open("matrices.txt", mode="r").read().split("\n")
+        m1 = ""
+        m2 = ""
+        c = 0
+        for line in file:
+            if line != "" and c == 0:
+                m1 += line 
+            else:
+                c += 1
+                m2 += line
+        matrix1, matrix2 = eval(m1), eval(m2)
+        A = len(matrix1)
```

Comment:
Upper case variables naming is associated with constant values. A -> a , etc. or even better to name variables more clear . Does A means rows ,  and B columns?

Reply:
Yes, A and B stand for columns and rows